### DISCUSSION TITLE
Editor/admins cannot edit navigation links

### DISCUSSION THREAD

**Discussion by brmscheiner** [requarks/wiki#3649] [Error / Bug Report]
**Describe the bug**
If a user is an "editor" and an "administrator", they get an error when trying to edit navigation links

![image](https://user-images.githubusercontent.com/9775178/77345454-e00ba880-6cf1-11ea-8439-0eaa0ca4dda0.png)

**Hacky solution**
Remove the user in question from the "editors" group

**Host Info**
 - OS: OS X
 - Wiki.js version: 2.1.113
 - Database engine: postgres 11.6

**Comment by Smankusors** [comment]
That's navigation links... not pages :confused:

**Comment by brmscheiner** [comment]
You're right! I'll edit the issue

**Comment by Smankusors** [comment]
btw are you just login and try to edit, or it's login from previous instance? Is this happens when you use incognito/private mode?
